src/SFA.DAS.EmployerAccounts.Web/Views/EmployerAccount/PayBillTriage.cshtml

```diff
@@ -0,0 +1,109 @@
+﻿@using SFA.DAS.EmployerAccounts.Web.Helpers
+@{
+    ViewBag.PageID = "pay-bill-triage";
+    ViewBag.Title = "Add a PAYE Scheme";
+    ViewBag.ShowNav = false;
+
+    ViewBag.GaData.Vpv = "/service/pay-bill-triage";
+    var hiddenClass = Model?.GetType().GetProperty("InError") != null ? "" : "das-hidden";
+    var errorClass = Model?.GetType().GetProperty("InError") != null ? "govuk-form-group--error" : "";
+}
+
+@if (Model?.GetType().GetProperty("InError") != null)
```

Comment:
Is there a reason we don't have a specific model for this view (or an interface?) so that the InError property evaluation is made straightforward? Probably a good reason I can't see.